polkadot/runtime/parachains/src/paras_inherent/tests.rs

```diff
@@ -2109,6 +2456,7 @@ mod sanitizers {
 			shared::Pallet::<Test>::add_allowed_relay_parent(
 				default_header().hash(),
 				Default::default(),
```

Comment:
I think we should also parameterise the `get_test_data_multiple_cores_per_para` function with the v2 candidates feature and if it's enabled, pass in the v2 receipts instead of the injected core index. This way we'll also cover the new code in the existing sanitisation tests

Reply:
done